### ISSUE TITLE
Windows Defender considers software dangerous

### ISSUE THREAD

**Issue by lestevam** [getferdi/ferdi#112]
When trying to install the program on windows 10 company Windows Defender warns about the danger of the software because it does not have a "publisher".

Even allowing windows defender installation, anti-virus does not allow installation for the same reason.

**Comment by vantezzen** [comment]
Thank you for reporting this error. What version of Ferdi are you trying to install?

**Comment by kytwb** [comment]
Maybe we need to _sign/notarize_ the app in a similar fashion to macOS but for Windows? https://www.electron.build/code-signing#windows

**Comment by kytwb** [comment]
@lestevam can you try the following steps in the meantime? https://stackoverflow.com/a/53685718/926721

>1. Right click on installer(.exe)
>2. Select properties option.
>3. Click on checkbox to check Unblock at the bottom of Properties.

**Comment by bittner** [comment]
LOL! Dear @microsoft, you suck. Good that Nadella announced it will slowly die.

**Comment by lestevam** [comment]
@kytwb I did realized the test in last version of Ferdi.
I did try o flow above but in my windows 10 this option isn´t showing.

**Comment by kytwb** [comment]
Closing in favour of https://github.com/getferdi/ferdi/issues/635 that has a more actionable title and is more recent 🙏

**Comment by TheCoinAxe** [comment]
I'm getting this anyone know why
![image](https://user-images.githubusercontent.com/65632488/114271278-45275e80-9a19-11eb-95ee-5c25c1008706.png)
